install.md

```diff
@@ -212,7 +307,8 @@ create a local configuration file (i.e. `dj_local_conf.json`) at the root of the
     "display.show_tuple_count": true,
     "custom": {
         "database.prefix": "<username_>",
-        "ephys_root_data_dir": "<~/data/ephys_root_data_dir>"
+        "ephys_root_data_dir": ["Full path to root directory of raw data",
+                                "Full path to root directory of processed data"]
```

Comment:
I think this will help clarify that there can be multiple
```suggestion
        "ephys_root_data_dir": ["Full first path to root directory of raw data",
                                "Full second path to root directory of raw data, optional",
                                "Full path(s) to root directory of processed data"]
```

Reply:
Thanks!